tests/integration/bootstrap/FeatureContext.php

```diff
@@ -80,4 +80,18 @@ public function disableDataImport()
 		$config = \Kohana::$config->load('features');
 		$config->set('data-import.enabled', false);
 	}
+
+	/** @BeforeScenario @cdnEnabled */
+	public function enableCdn()
```

Comment:
@rjmackay @crcommons If it doesn't work then dropping it seems fine. Out of curiosity why doesn't it work?